<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordDialog</class>
 <widget class="QDialog" name="ForgotPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forgot Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2c3e50; margin-bottom: 4px;</string>
     </property>
     <property name="text">
      <string>ðŸ”“  Forgot Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblDesc">
     <property name="styleSheet">
      <string notr="true">color: #7f8c8d; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Enter your username and answer the security question.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblUsername">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Enter username...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVerifyUser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#3498db; 
    color:white; 
    border-radius:6px; 
    font-size:12px; 
    padding: 0 12px; 
}
QPushButton:hover { 
    background:#2980b9; 
}</string>
       </property>
       <property name="text">
        <string>Verify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblQuestion">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Security question:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblQuestionText">
     <property name="styleSheet">
      <string notr="true">color: #2980b9; 
font-style: italic; 
font-size: 12px; 
padding: 6px 10px; 
background:#eaf4fd; 
border-radius:6px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblAnswer">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Answer:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditAnswer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter your answer...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#2ecc71; 
    color:white; 
    border-radius:6px;
    font-size:13px; 
    font-weight:bold; 
    padding: 0 16px; 
}
QPushButton:hover { 
    background:#27ae60; 
}</string>
       </property>
       <property name="text">
        <string>âœ”  Reset Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#ecf0f1; 
    color:#7f8c8d; 
    border-radius:6px;
    font-size:13px; 
    padding: 0 16px; 
}
QPushButton:hover { 
    background:#d5dbdb; 
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ForgotPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
